packages/node_modules/@webex/webex-core/src/config.js

```diff
@@ -19,7 +19,46 @@ export default {
   AlternateLogger: undefined,
   credentials: new CredentialsConfig(),
   services: {
-    u2c: process.env.U2C_SERVICE_URL || 'https://u2c.wbx2.com/u2c/api/v1'
+    /**
+     * A list of services that are available prior to catalog collection.
+     *
+     * @type {Object}
+     */
+    discovery: {
+      /**
+       * The hydra discovery url.
+       *
+       * @type {string}
+       */
+      hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
+
+      /**
+       * The u2c discovery url
+       *
+       * @type {string}
+       */
+      u2c: process.env.U2C_SERVICE_URL || 'https://u2c.wbx2.com/u2c/api/v1'
+    },
+
+    /**
+     * When true, enable whitelisted domain validation.
+     *
+     * @type {boolean}
+     */
+    validateDomains: process.env.VALIDATE_DOMAINS === 'true' || false,
+
+    /**
+     * Contains a list of whitelisted host addresses.
+     *
+     * @type {Array<string>}
+     */
+    whitelist: [
```

Comment:
Could we rename this to a more inclusive name like `allowedHosts`?

Reply:
This can be done and makes more sense, let me get to this change.